src/native/managed/cdac/Microsoft.Diagnostics.DataContractReader.Contracts/Contracts/Loader_1.cs

```diff
@@ -196,6 +200,76 @@ bool ILoader.TryGetLoadedImageContents(ModuleHandle handle, out TargetPointer ba
         return true;
     }
 
+    private static uint AlignUp(uint value, uint alignment)
+    {
+        if (alignment == 0)
```

Comment:
Perhaps a stronger check here?
```suggestion
        if (!BitOperations.IsPow2(alignment))
```

Reply:
I did not realize I had that check, Copilot must have snuck it in somehow… The original code does not have this, so I think we can do without it.